### PR TITLE
Introduced config files for RSV and a new lablog for results in viralrecon template

### PR DESCRIPTION
**Pull request by Shettland**
Several config files were included in viralrecon template: [multiqc_config.yml](https://github.com/Shettland/buisciii-tools/blob/develop/bu_isciii/templates/viralrecon/DOC/multiqc_config.yml), [viralrecon_rsv.config](https://github.com/Shettland/buisciii-tools/blob/develop/bu_isciii/templates/viralrecon/DOC/viralrecon_rsv.config) and [viralrecon_rsv_params.yml](https://github.com/Shettland/buisciii-tools/blob/develop/bu_isciii/templates/viralrecon/DOC/viralrecon_rsv_params.yml).

Some implementations were introduced into the results [lablog](https://github.com/Shettland/buisciii-tools/blob/develop/bu_isciii/templates/viralrecon/RESULTS/lablog) to work for more than one reference, and a new script [excel_generator.py](https://github.com/Shettland/buisciii-tools/blob/develop/bu_isciii/templates/viralrecon/RESULTS/excel_generator.py) was included to process the results. The old lablog was deleted.

### CODE REVIEW

File: bu_isciii/templates/viralrecon/RESULTS/lablog
Comment (svarona) [inline bu_isciii/templates/viralrecon/RESULTS/lablog:1]: El fichero de RESULTS se tiene que llamar viralrecon_lablog o viralrecon results porque puede haber más de un servicio que se esté entregando y si todos se llaman lablog se sobre-escriben, igual que en ANALISIS, cambiale el nombre

File: bu_isciii/templates/viralrecon/RESULTS/excel_generator.py
Comment (svarona) [inline bu_isciii/templates/viralrecon/RESULTS/excel_generator.py:1]: Puedes añadir a los templates un template del samples_ref de ANALYSIS y un template del reference_file necesario para lanzar el excel_generator?

**Review by svarona — COMMENTED** [review]
Revisa los comentarios porfa!

File: bu_isciii/templates/viralrecon/RESULTS/lablog
Comment (Shettland) [reply bu_isciii/templates/viralrecon/RESULTS/lablog:1]: Entendido, ahora lo cambio.

**Review by svarona — APPROVED** [review]

**Pull request merged**